pages/_app.tsx

```diff
@@ -133,7 +141,12 @@ export default function App(props: AppProps) {
       />
 
       <ChakraProvider theme={theme}>
-        <Component {...pageProps} />
+        <Container maxW="100%" px={{ base: 4, md: 6 }} pt="6">
```

Comment:
`<Navbar>`, `<RegisterForm>` and `<Footer>` moved to `_app.tsx` so these are common to all pages (if we add others in the future!)